src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import { BADGE } from '@geometricpanda/storybook-addon-badges';
```

Comment:
Is it worth having stories & tests for this if it's covered by the `InputField` stories? I guess my mental model is this is a subcomponent like `CheckboxInput` (or maybe better example is `PopoverContainer`, which is also used by multiple components). Worry that it's confusing to show stories for a component that's only for internal use

Reply:
That's a good point. I was being cautious about removing tests / stories but do agree that it's not worth having these. Will remove